### ISSUE TITLE
Update Gradle to 6.x?

### ISSUE THREAD

**Issue by vlsi** [myniva/gradle-s3-build-cache#41]
Gradle 4.0 is somewhat old nowdays.

**Comment by vlsi** [comment]
shipkit does not support Gradle 6 yet: https://github.com/mockito/shipkit/issues/876

**Comment by myniva** [comment]
Yes, I was thinking about that as well. The only thing we should keep in mind is backwards compatibility. I want the plugin to stay compatible with Gradle >= 4.0.

As far as I can tell, this shouldn't be a problem, as the Build Cache API seems to be stable since 3.5.
